### PR TITLE
Shovel messages from files.verified

### PR DESCRIPTION
**Pull request by jonandernovella**
### Describe the pull request:

- [ ] Bug fix
- [x] Functional change
- [ ] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Pull request long description:

We should now shovel messages from the verified queue as well as of completed.
### Changes made:

The MQ configuration now points to verified.

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -198,7 +198,7 @@ cat > "/var/lib/rabbitmq/advanced.config" <<EOF
               {'queue.bind',
                 [{exchange, <<"lega">>},
                   {queue, <<>>},
-                  {routing_key, <<"completed">>}
```
Comment (dtitov) [inline entrypoint.sh]: This doesn't seem to be correct. If I understood your changes correctly, in the existing `cega_completion` shovel you just replaced the routing keys from `files.completed` to `files.verified`. But we still need to shovel completion message to CEGA (the last step in the data flow: https://localega.readthedocs.io/en/latest/_images/amqp-ping-pong.png). So we can't just change `cega_completion` - we need to keep it as it is now, but additionally introduce similar shovel called `cega_verification` that will separately shovel the verification messages.

File: entrypoint.sh [hunk 2]
```diff
@@ -210,7 +210,7 @@ cat > "/var/lib/rabbitmq/advanced.config" <<EOF
               {declarations, []},
               {publish_properties, [{delivery_mode, 2}]},
               {publish_fields, [{exchange, <<"cega">>},
-                {routing_key, <<"files.completed">>}
```
Comment (dtitov) [inline entrypoint.sh]: See above.

File: entrypoint.sh [hunk 3]
```diff
@@ -271,7 +271,7 @@ cat > "/var/lib/rabbitmq/definitions.json" <<EOF
       "arguments": {}
     },
     {
-      "name": "files.completed",
```
Comment (dtitov) [inline entrypoint.sh]: See above.

**Review by dtitov — CHANGES_REQUESTED** [review]

File: entrypoint.sh (hunk 1 above)
Comment (blankdots) [reply entrypoint.sh]: so we just add another shovel for `file.verified`

**Review by blankdots — APPROVED** [review]

**Review by dtitov — APPROVED** [review]

**Pull request merged**